### ISSUE TITLE
Stack trace when using default-param transform

### ISSUE THREAD

**Issue by laino** [lebab/lebab#279]
```
/home/lain/.npm/node_modules/lebab/lib/transform/defaultParam/index.js:123
        throw _iteratorError;
        ^

TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined
    at findDefaults (/home/lain/.npm/node_modules/lebab/lib/transform/defaultParam/index.js:104:32)
    at transformDefaultParams (/home/lain/.npm/node_modules/lebab/lib/transform/defaultParam/index.js:48:26)
    at Controller.enter (/home/lain/.npm/node_modules/lebab/lib/transform/defaultParam/index.js:11:9)
    at Controller.__execute (/home/lain/.npm/node_modules/estraverse/estraverse.js:397:31)
    at Controller.replace (/home/lain/.npm/node_modules/estraverse/estraverse.js:637:27)
    at Object.replace (/home/lain/.npm/node_modules/estraverse/estraverse.js:718:27)
    at Object.replace (/home/lain/.npm/node_modules/lebab/lib/traverser.js:59:33)
    at exports.default (/home/lain/.npm/node_modules/lebab/lib/transform/defaultParam/index.js:8:23)
    at /home/lain/.npm/node_modules/lebab/lib/Transformer.js:69:11
    at Array.forEach (<anonymous>)
```

**Comment by uniibu** [comment]
This has already been fixed by PR https://github.com/lebab/lebab/pull/278 But a patch has not yet been released yet. Until then, what you can do is make sure `default-param` transform is first on the list of transform options before `arrow-return`

**Comment by nene** [comment]
Sorry for the delay. The fix is now out in Lebab 3.0.3.
